src/components/Inngest.ts

```diff
@@ -89,7 +91,7 @@ export class Inngest<Events extends Record<string, EventPayload>> {
    */
   constructor({
     name,
-    eventKey = process.env[envKeys.EventKey],
```

Comment:
This was a bug unrelated to this PR, but needed fixing to test! We can't use `process.env` like this, as we're in shared code here.